department-of-veterans-affairs/next-build
# update storybook deps

## Description
Storybook static output has been failing on tugboat, updating the dependencies and it builds successfully.

## Testing done
`yarn storybook:build`

the tugboat storybook preview should render without error

## Screenshots
Fixed:
```
yarn storybook:build
@storybook/cli v7.6.3

info => Cleaning outputDir: /out/storybook-static
info => Loading presets
info => Building manager..
info => Manager built (67 ms)
info => Building preview..
info => Copying static files: /Users/heffner/dev/next-build/node_modules/@storybook/manager/static at /Users/heffner/dev/next-build/out/storybook-static/sb-common-assets
info Using Babel compiler
info Addon-docs: using MDX2
info => Using implicit CSS loaders
Using environment variables from: envs/.env.local
info => Using default Webpack5 setup
10% building 0/1 entries 1/1 dependencies 0/1 modulesinfo => Copying static files: /Users/heffner/dev/next-build/public at /Users/heffner/dev/next-build/out/storybook-static/
WARN asset size limit: The following asset(s) exceed the recommended size limit (244 KiB).
WARN This can impact web performance.
WARN Assets:
WARN   3426.fbd50044.iframe.bundle.js (583 KiB)
WARN   497.63949dfb.iframe.bundle.js (711 KiB)
WARN   4170.e1466f9c.iframe.bundle.js (952 KiB)
WARN entrypoint size limit: The following entrypoint(s) combined asset size exceeds the recommended limit (244 KiB). This can impact web performance.
WARN Entrypoints:
WARN   main (993 KiB)
WARN       runtime~main.92a925f8.iframe.bundle.js
WARN       4170.e1466f9c.iframe.bundle.js
WARN       main.57cd201c.iframe.bundle.js
WARN
info => Preview built (6.38 s)
info => Output directory: /Users/heffner/dev/next-build/out/storybook-static
```

Before:
```
yarn storybook:build
@storybook/cli v7.5.1

info => Cleaning outputDir: /out/storybook-static
info => Loading presets
info => Building manager..
info => Manager built (73 ms)
info => Building preview..
info => Compiling preview..
info => Copying static files: /Users/heffner/dev/next-build/node_modules/@storybook/manager/static at /Users/heffner/dev/next-build/out/storybook-static/sb-common-assets
info Addon-docs: using MDX2
info => Using implicit CSS loaders
WARN 🚨 Unable to index files:
WARN - ./src/stories/Introduction.stories.mdx: indexer.createIndex is not a function
WARN - ./src/templates/components/expandableText/expandableText.stories.mdx: indexer.createIndex is not a function
WARN - ./src/templates/components/table/table.stories.mdx: indexer.createIndex is not a function
WARN - ./src/templates/globals/banners/banner/banner.stories.mdx: indexer.createIndex is not a function
WARN - ./src/templates/globals/banners/facilityBanner/facilityBanner.stories.mdx: indexer.createIndex is not a function
WARN - ./src/templates/globals/banners/promoBanner/promoBanner.stories.mdx: indexer.createIndex is not a function
WARN 🚨 Unable to index files:
WARN - ./src/stories/Introduction.stories.mdx: indexer.createIndex is not a function
WARN - ./src/templates/components/expandableText/expandableText.stories.mdx: indexer.createIndex is not a function
WARN - ./src/templates/components/table/table.stories.mdx: indexer.createIndex is not a function
WARN - ./src/templates/globals/banners/banner/banner.stories.mdx: indexer.createIndex is not a function
WARN - ./src/templates/globals/banners/facilityBanner/facilityBanner.stories.mdx: indexer.createIndex is not a function
WARN - ./src/templates/globals/banners/promoBanner/promoBanner.stories.mdx: indexer.createIndex is not a function
Error: Unable to index files:
- ./src/stories/Introduction.stories.mdx: indexer.createIndex is not a function
- ./src/templates/components/expandableText/expandableText.stories.mdx: indexer.createIndex is not a function
- ./src/templates/components/table/table.stories.mdx: indexer.createIndex is not a function
- ./src/templates/globals/banners/banner/banner.stories.mdx: indexer.createIndex is not a function
- ./src/templates/globals/banners/facilityBanner/facilityBanner.stories.mdx: indexer.createIndex is not a function
- ./src/templates/globals/banners/promoBanner/promoBanner.stories.mdx: indexer.createIndex is not a function
    at StoryIndexGenerator.getIndex (./node_modules/@storybook/core-server/dist/index.js:60:3746)
    at async extractStoriesJson (./node_modules/@storybook/core-server/dist/index.js:51:4620)
    at async Promise.all (index 3)
    at async buildStaticStandalone (./node_modules/@storybook/core-server/dist/index.js:99:2252)
    at async withTelemetry (./node_modules/@storybook/core-server/dist/index.js:103:3903)
    at async build (./node_modules/@storybook/cli/dist/generate.js:473:1059)
    at async Command.<anonymous> (./node_modules/@storybook/cli/dist/generate.js:476:125)
Using environment variables from: envs/.env.local
info => Using default Webpack5 setup
? Would you like to help improve Storybook by sending anonymous crash reports? › (Y/n)info => Copying static files: /Users/heffner/dev/next-build/public at /Users/heffner/dev/next-build/out/storybook-static/
info => Preview built (5.48 s)
WARN Critical dependency: require function is used in a way in which dependencies cannot be statically extracted
WARN Critical dependency: require function is used in a way in which dependencies cannot be statically extracted
WARN asset size limit: The following asset(s) exceed the recommended size limit (244 KiB).
WARN This can impact web performance.
WARN Assets:
WARN   3426.fbd50044.iframe.bundle.js (583 KiB)
WARN   9569.2f632969.iframe.bundle.js (675 KiB)
WARN   1439.9ff470b1.iframe.bundle.js (860 KiB)
WARN entrypoint size limit: The following entrypoint(s) combined asset size exceeds the recommended limit (244 KiB). This can impact web performance.
WARN Entrypoints:
WARN   main (899 KiB)
WARN       runtime~main.d37b0ef6.iframe.bundle.js
WARN       1439.9ff470b1.iframe.bundle.js
WARN       main.9df3cce1.iframe.bundle.js
WARN
info => Preview built (5.6 s)
```

## QA steps

Check the main tugboat preview storybook: https://storybook-kqjsor0i3mjdwsy9gojxhhyvdzh0wubb.tugboat.vfs.va.gov/ 
Verify that it is broken: 
![Screenshot 2023-12-04 at 8 30 46 AM](https://github.com/department-of-veterans-affairs/next-build/assets/11279744/ed620980-e1b5-4be1-b006-8c179c1f6210)

Check this PR's storybook preview: https://storybook-22uq3wnf1kddlbdktg9eo3raomyz5def.tugboat.vfs.va.gov/
Verify that is it not: 
![Screenshot 2023-12-04 at 8 30 50 AM](https://github.com/department-of-veterans-affairs/next-build/assets/11279744/c2132881-6a1e-4e84-9610-9d2bf9443fb8)


```[tasklist]
- [x] Storybook on tugboat generates without errors
```
